frontend/src/app/animals-list/animal-filter/animal-filter.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="animal-filter">
+  <form [formGroup]="filter" (ngSubmit)="onSubmit()">
+    <select class="animal-filter__select-type" formControlName="filter_type">
```

Comment:
Add `animal-filter__select` class to this component (along with `animal-filter__select-type`). It prevents styles duplicate in scss file